tracer/src/Datadog.Trace.Manual/CorrelationIdentifier.cs

```diff
@@ -57,6 +57,19 @@ public static ulong TraceId
         }
     }
 
+    /// <summary>
+    /// Gets the trace's unique 128-bit identifier as a hexadecimal string
+    /// of length 32, padded with zeros to the left if the identifier is 64-bit.
+    /// </summary>
+    /// <returns>The id of the active trace. If there is no active trace, returns null.</returns>
+    public static string? RawTraceId
```

Comment:
A bit nitpicky:
I feel like this makes the API a bit confusing, `TraceId` returns a different value and type than `RawTraceId` and I'm wondering if `RawTraceId` would be more useful to most as it can both represent the 64-bit and 128-bit versions.

Would it be too large of a breaking change (or just not make sense) to change the other `TraceId` property to go from `ulong` to `string?` and always return the 128-bit hex string?

Reply:
I'd say that I'm fine with it being left as is then